packages/abi-coder/src/abi-coder.ts

```diff
@@ -195,9 +195,7 @@ export default class AbiCoder {
 
     const coders = nonEmptyTypes.map((type) => this.getCoder(type));
     if (nonEmptyTypes[0] && nonEmptyTypes[0].type === 'raw untyped slice') {
-      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-      // @ts-ignore
-      coders[0].length = bytes.length / 8;
+      (coders[0] as ArrayCoder<any>).length = bytes.length / 8;
```

Comment:
Lint is complaining about `ArrayCoder<any>`: _"Unexpected any. Specify a different type_"

Would it work with `ArrayCoder<unknown>`?

If not, we could use an `eslint` ignore comment/rule.

> **Note** @FuelLabs/sdk-ts — About the `any` rule:
> 
> _A while ago, @luizstacio suggested treating these `any` warnings as errors, which makes sense, IMO. We could all start familiarizing ourselves with the idea. Followup issue; feel free to vote up/down_:
> - https://github.com/FuelLabs/fuels-ts/issues/821)